scene/gui/graph_edit.cpp

```diff
@@ -1244,6 +1292,7 @@ void GraphEdit::gui_input(const Ref<InputEvent> &p_ev) {
 						gn->set_drag(false);
 						if (gn->is_comment()) {
 							_set_drag_comment_enclosed_nodes(gn, comment_enclosed_nodes, false);
+							_reorder_comment_nodes(gn);
```

Comment:
I'm a bit confused by the order of operations. In the below snippet we update enclosed nodes after the reorder, and here we do before. It's a bit hard to follow the logic, so can you tell if this is intentional?